Civi/Test/Legacy/CiviTestListener.php

```diff
@@ -28,7 +28,7 @@ class CiviTestListener extends \PHPUnit_Framework_BaseTestListener {
   private $cache = [];
 
   /**
-   * @var \CRM_Core_Transaction|NULL
+   * @var \CRM_Core_Transaction|null
```

Comment:
This feels decidedly quirky in light of https://www.drupal.org/docs/develop/standards/coding-standards#s-constants , but it's what the linter wants.